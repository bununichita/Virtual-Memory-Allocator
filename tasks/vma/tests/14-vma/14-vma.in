ALLOC_ARENA 68719476836
ALLOC_BLOCK 17179869194 8589934592
PMAP
FREE_BLOCK 17179869194
DEALLOC_ARENA
