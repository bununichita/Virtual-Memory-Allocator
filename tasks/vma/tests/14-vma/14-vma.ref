Total memory: 0x1000000064 bytes
Free memory: 0xE00000064 bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 1

Block 1 begin
Zone: 0x40000000A - 0x60000000A
Miniblock 1:		0x40000000A		-		0x60000000A		| RW-
Block 1 end
