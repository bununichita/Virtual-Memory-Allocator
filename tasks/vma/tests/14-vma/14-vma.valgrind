==36268== Memcheck, a memory error detector
==36268== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==36268== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==36268== Command: /home/nicky/Desktop/SDA/Virtual-Memory-Allocator/vma
==36268== Parent PID: 36267
==36268== 
==36268== 
==36268== HEAP SUMMARY:
==36268==     in use at exit: 0 bytes in 0 blocks
==36268==   total heap usage: 10 allocs, 10 frees, 8,377 bytes allocated
==36268== 
==36268== All heap blocks were freed -- no leaks are possible
==36268== 
==36268== For lists of detected and suppressed errors, rerun with: -s
==36268== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
