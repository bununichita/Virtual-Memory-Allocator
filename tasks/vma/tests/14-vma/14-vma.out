Total memory: 1000000064 bytes
Free memory: -3996 bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 1

Block 1 begin
Zone: 40000000A - 40000100A
Miniblock 1:	40000000A	-	40000100A	| RW-
Block 1 end
